plugins/auth-auth0/src/static/client/auth-provider/ra-auth-auth0.ts

```diff
@@ -0,0 +1,95 @@
+import { Auth0Client } from "@auth0/auth0-spa-js";
+import { AuthProvider, UserIdentity } from "react-admin";
+
+export const PreviousLocationStorageKey = "@react-admin/nextPathname";
+
+export const client = new Auth0Client({
+  domain: process.env.REACT_APP_AUTH0_DOMAIN || "",
+  clientId: process.env.REACT_APP_AUTH0_CLIENT_ID || "",
+  cacheLocation: "localstorage",
+  authorizationParams: {
+    audience: process.env.REACT_APP_AUTH0_AUDIENCE,
+    scope: "openid profile email",
+  },
+  useRefreshTokens: true,
+});
+
```

Comment:
<img width="1125" alt="image" src="https://github.com/amplication/plugins/assets/2861984/a33f995c-f16b-4249-9bd8-51ac7e932d07">

Is the login supposed to work?

Reply:
Yes, its working.